### PR TITLE
Create pages

### PR DESCRIPTION
**Pull request by Alusp**
## I implemented the following in this PR:

- [ ] Set up the repository and tools.
- [ ] Use Postgres as your database.
- [ ] Use devise for authentication.
- [ ] Validate all user input to make sure that anyone with bad intentions cannot compromise your app.
- [ ] Use a view template engine of my choice (.erb).
- [ ] The project should be deployed and accessible online.
- [ ] Your database schema reflect the following structure:
- [ ] Create my models according to the ERD diagram.
- [ ] Implement the authentication and authorization.
- [ ] Create the page to add a category.
- [ ] Create the home page.
- [ ] Create the page to add a transaction.
- [ ] Create the transactions page.
- [ ] Make sure my app is tested adequately.
- [ ] Deploy the project, and test for final details.
- [ ] Record a video for your project.
- [ ] Create a good README and PR description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,29 +35,29 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [your_project_name] <a name="BUDGET APP"></a>
```
Comment (KDlamini) [inline README.md]: Kindly fix your project name, I still see  `[your_project_name] ` as your project title.

File: README.md [hunk 2]
```diff
@@ -94,137 +72,111 @@ After you're finished please remove all the comments and instructions!
...
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+>Demonstration project.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://www.loom.com/share/46f73bd7b34749ddbe4ce84357a37661)
```
Comment (KDlamini) [inline README.md]: Kindly rename your video link to `presentation video` or similar. There are two links named "Live Demo Link" on your READMe which causes confusion as to which is which.

File: app/views/groups/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                <% group.group_purchases.each { |e| @total += e.purchase.amount }%>
+                <p class="text-end col-3">Total: $<%= @total %></p>
+              </div>
+            <% end %>
+          </div>
+        <% else %>
+          <p class="p-2">No category added yet</p>
+        <% end %>
+        <div class="text-light footer d-flex justify-content-center p-3">
+          <%= link_to "Add new category", new_group_path, class: 'btn btn-outline-dark' %>
```
Comment (KDlamini) [inline app/views/groups/index.html.erb]: Kindly make sure that you follow the design guidelines (colors, typography, and layout) for your buttons.

image

Your design:
image

Required design:
image

Same thing for your transactions/purchases button:
image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @Alusp 🙋‍♀️,  

Great work so far! 💯 
You have done a great job creating a working Budget app using Ruby on Rails. 🎊 
There's always room for improvement. There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Database, Models, controllers, routes, and views set up correctly
✔️  User authentication using Devise implemented
✔️  User inputs validated
✔️  App is secured from n + 1 problems
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alusp** [comment]
Alright @KDlamini , Thank you very much for the review, fixing it right away.

**Review by uwadonat — APPROVED** [review]
## Hi @Alusp,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- No linter errors ✔️
- Good pull request message ✔️

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Alusp** [comment]
Thank you very much for the Review @uwadonat

**Pull request merged**